resources/assets/enderio/lang/ru_RU.lang

```diff
@@ -1251,14 +1252,62 @@ enderio.tooltip.uid=UID:
 enderio.tooltip.anvilupgrades=Улучшения на наковальне:
 enderio.waila.config.hiddenfacades=Скрытые фасады
 
-enderio.config.capacitor.alloy_smelter_power_buffer=Alloy Smelter: Internal power buffer size (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.alloy_smelter_power_intake=Alloy Smelter: Maximum power intake per tick (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.alloy_smelter_power_use=Alloy Smelter: Power use per tick (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.buffer_power_buffer=Energy/Omni Buffer: Internal power buffer size (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.buffer_power_intake=Energy/Omni Buffer: Maximum power intake per tick (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.crafter_power_buffer=Crafter: Internal power buffer size (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.crafter_power_intake=Crafter: Maximum power intake per tick (will be scaled by the used upgrade capacitor)
-enderio.config.capacitor.crafter_ticks=Crafter: Ticks per crafting operation (will be scaled by the used upgrade capacitor)
+enderio.config.capacitor.alloy_smelter_power_buffer=Завод сплавов: Размер внутреннего буфера энергии (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.alloy_smelter_power_intake=Завод сплавов: Максимальная энергия, потребляемая за тик (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.alloy_smelter_power_use=Завод сплавов: Power use per tick (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.buffer_power_buffer=Энергетический/универсальный буфер: Размер внутреннего буфера энергии (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.buffer_power_intake=Энергетический/универсальный буфер: Максимальная энергия, потребляемая за тик (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_power_buffer=Создатель: Размер внутреннего буфера энергии (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_power_intake=Создатель: Максимальная энергия, потребляемая за тик (будет масштабироваться при использовании конденсатора)
+enderio.config.capacitor.crafter_ticks=Создатель: Тиков за операцию крафта (будет масштабироваться при использовании конденсатора)
 
-// There's a 1% change that a soul vial containing a pig is names this:
+// There's a 1% change that a soul vial containing a pig is named this:
```

Comment:
Didn't I fix that one already?

However, I don't care about comments bein in sync between the languages. In fact, it's a bad thing. Put your own comments in. If you had to look up how a string is used in-game, write it down for the next person to work on the translation. If you translated some word oddly, write down why (e.g. same wording as OtherMod).

And BTW, to check if your translation is complete, or has dead lines, run "gradle checkTranslations"